packages/studio-base/src/panels/ThreeDeeRender/renderables/Axis.ts

```diff
@@ -126,14 +126,14 @@ export class Axis extends THREE.Object3D {
   }
 }
 
-function standardMaterial(materialCache: MaterialCache): THREE.MeshStandardMaterial {
-  return materialCache.acquire(
-    StandardColor.id(COLOR_WHITE),
-    () => StandardColor.create(COLOR_WHITE),
-    StandardColor.dispose,
-  );
-}
-
-function releaseStandardMaterial(materialCache: MaterialCache): void {
-  materialCache.release(StandardColor.id(COLOR_WHITE));
+function standardMaterial(color: ColorRGBA): THREE.MeshStandardMaterial {
```

Comment:
```suggestion
function makeStandardMaterial(color: ColorRGBA): THREE.MeshStandardMaterial {
```
Might make it clearer that this is allocating a new material instance